Description: Don't check if gmp is available.
 Since gmp has been multiarch-ified recently libgmp.so is to be found in
 /usr/lib/<triplet>. But setup.py only checks /lib and /usr/lib.
Origin: Sebastian Ramacher <s.ramacher@gmx.at>
Last-Update: 2011-09-24

diff --git a/setup.py b/setup.py
index 37aedef..88c87e9 100644
--- a/setup.py
+++ b/setup.py
@@ -156,12 +156,6 @@ class PCTBuildExt (build_ext):
         if self.compiler.compiler_type == 'msvc':
             self.compiler.include_dirs.insert(0, "src/inc-msvc/")

-        # Detect libgmp and don't build _fastmath if it is missing.
-        lib_dirs = self.compiler.library_dirs + ['/lib', '/usr/lib']
-        if not (self.compiler.find_library_file(lib_dirs, 'gmp')):
-            print >>sys.stderr, "warning: GMP library not found; Not building Crypto.PublicKey._fastmath."
-            self.__remove_extensions(["Crypto.PublicKey._fastmath"])
-
     def __remove_extensions(self, names):
         """Remove the specified extension from the list of extensions to build"""
         i = 0
